nix/sources.json

```diff
@@ -36,15 +36,39 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "nixpkgs": {
-        "branch": "master",
+        "branch": "nixpkgs-unstable",
```

Comment:
nop, sources.json is updated through `niv` manually.

Reply:
Your comment is worth noting. 

When you have some time, please document related contents in Notion or something similar. 
It could be very useful for someone working on Nix-related tasks in the future.

cc. @dongsam